pkg/cleaner/jobcontroller.go

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright 2019 OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+This pkg is inspired from the deleter pkg in local-static-provisioner
+in kubernetes-sigs
+	https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/pkg/deleter
+*/
+
+package cleaner
+
+import (
+	"context"
+	"github.com/openebs/node-disk-manager/cmd/ndm_daemonset/controller"
+	"github.com/openebs/node-disk-manager/pkg/apis/openebs/v1alpha1"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strconv"
+)
+
+const (
+	// JobContainerName is the name of the cleanup job container
+	JobContainerName = "cleaner"
+	// JobNamePrefix is the prefix for the cleanup job name
+	JobNamePrefix = "cleanup-"
+	// JobImageName is the image to be used for the cleanup job container
+	JobImageName = "busybox"
+	// BDLabel is the label set on the job for identification of the BD
+	BDLabel = "blockdevice"
+	// BlockCleanerCommand is the command used to clean raw block
+	BlockCleanerCommand = "dd"
```

Comment:
@kmova  Should we go with wipefs? `dd` is taking so much time to cleanup

Reply:
Let us go with dd for now.